pkgs/applications/networking/modem-manager-gui/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, buildEnv, pkgconfig, python3, fetchhg, gtk3-x11, glib, gdbm, gtkspell3, itstool, libappindicator-gtk3, meson, ninja, perldevelPackages }:
+
+stdenv.mkDerivation rec {
+  name = "modem-manager-gui";
+  version = "0.4.17";
+  rev = "v${version}";
+
+  src = fetchhg {
+    url = https://linuxonly@bitbucket.org/linuxonly/modem-manager-gui;
+    rev = "8bcf842";
+    sha256 = "04p08gzvi068bd3slbnaghajwzp4h5jacib640k5shngqimkl84p";
+  };
+
+  buildInputs = [
+    pkgconfig
+    gtk3-x11
+    glib
+    gdbm
+    gtkspell3
+
+    python3
+    perldevelPackages.Po4a
+    itstool
+    libappindicator-gtk3
+  ];
+
+  preConfigure = ''
+    sed -i 's|! test -e "/usr/include/gdbm/gdbm.h"|false|' configure
+    sed -i "s|/etc/NetworkManager/dispatcher.d|$out/etc/NetworkManager/dispatcher.d|" src/scripts/Makefile
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://linuxonly.ru/page/modem-manager-gui;
+    description = "A simple GTK+ based GUI compatible with Modem manager, Wader and oFono system services able to control EDGE/3G/4G broadband modem specific functions. You can check balance of your SIM card, send or receive SMS messages, control mobile traffic consumption and more.";
```

You might want to wrap this using `''`

```suggestion
    description = ''
      A simple GTK+ based GUI compatible with Modem manager, Wader and oFono
      system services able to control EDGE/3G/4G broadband modem specific
      functions. You can check balance of your SIM card, send or receive SMS
      messages, control mobile traffic consumption and more.
    '';
```